src/app/patient-summary/components/resources/Patient/ConnectionDetails.tsx

```diff
@@ -0,0 +1,64 @@
+import { ArrayCaption } from "@/app/components/typography/ArrayCaption";
+import {
+  TableContainer,
+  Paper,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+} from "@mui/material";
+
+export default function ConnectionDetails({ patientInfo }) {
+  const addressRows = patientInfo.address.map((addressInfo) => {
+    return {
+      use: addressInfo.use,
+      city: addressInfo.city,
+      state: addressInfo.state,
+      postalCode: addressInfo.postalCode,
+      country: addressInfo.country,
+      line: addressInfo.line.map((lineInfo) => lineInfo),
+    };
+  });
+
+  const phoneRows = patientInfo.telecom.map((telecomInfo) => {
+    return {
+      use: telecomInfo.use,
+      system: telecomInfo.system,
+      value: telecomInfo.value,
+    };
+  });
+
+  return (
+    <>
+      <ArrayCaption caption="Connection Details" />
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 650 }} size="small">
+          <TableHead>
+            <TableRow>
+              <TableCell>Type</TableCell>
+              <TableCell align="right">Info</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {addressRows.map((row, idx) => (
+              <TableRow key={idx}>
+                <TableCell>{row.use} Address</TableCell>
+                <TableCell align="right">{`${row.line},${row.city}, ${row.postalCode}, ${row.country}`}</TableCell>
```

Comment:
Not critical for now, but something to consider/keep in mind is how this will render if these values arent populated within the FHIR resource. Happy to skip this for now and we can implement these checks another time

Reply:
Yes that's right, I'll keep that in mind for the next PR.